src/index.js

```diff
@@ -0,0 +1,32 @@
+import './style.css';
+import {
+  addEventsToLabels, addEventsCheckBoxes, addEventsNewTasks, addEventclearAllButton,
+  deleteFromIcons,
+} from './utils.js';
+
+window.onload = () => {
```

Comment:
- [ ] From the project requirement you are expected to have a written array of tasks, not loaded from the local storage, in your code that is presented on the page. Kindly take note of that and add this array and ensure its contents are displayed on the browser.

  ![image](https://github.com/alex1779/to-do-list-v3/assets/81468675/77074498-e1cc-4177-9f22-e8532ae22f7d)